src/components/tooltips/APRTooltip/components/StakingBreakdown.vue

```diff
@@ -85,7 +87,14 @@ const breakdownItems = computed((): Array<any> => {
     if (isMinMaxSame.value) {
       items.push(['BAL', minBalAPR.value]);
     }
-    items.push(['Rewards', rewardTokensAPR.value]);
+
+    let rewardAprTokenSymbol = '';
+    const rewardAprTokens = apr.value?.rewardAprs.breakdown;
+    if (rewardAprTokens) {
+      rewardAprTokenSymbol = getToken(Object.keys(rewardAprTokens)[0])?.symbol;
```

Comment:
What if there are more than 1 reward tokens? It's possible that there could be several.

Reply:
We should get it to items.push for every token in rewardAprTokens.

Reply:
how we should display them?

Reply:
hmm this is set up only to display the total atm. You will need to replace that with a v-for and just list all the reward tokens here: https://github.com/balancer-labs/frontend-v2/blob/c943c8e48b66764e7fc58015bce9b9262c955045/src/components/tooltips/APRTooltip/components/StakingBreakdown.vue#L133-L138

Reply:
Ahh wait, that's not quite right

Reply:
Instead of doing this you just need to do a loop through each of the https://github.com/balancer-labs/frontend-v2/blob/c943c8e48b66764e7fc58015bce9b9262c955045/src/components/tooltips/APRTooltip/components/StakingBreakdown.vue#L97
You need to do a loop through `rewardAprTokens` and push a new item for each token in the map.

Reply:
fixed